runtime/onert/core/src/exec/JSONExecTime.cc

```diff
@@ -135,6 +135,9 @@ void JSON::printOperation(const std::map<uint32_t, int64_t> &operation_info,
   stream.seekp(-2, std::ofstream::end);
 }
 
+using MeasurementData = std::unordered_map<
+  const backend::Backend *,
+  std::unordered_map<std::string, std::unordered_map<bool, std::map<uint32_t, int64_t>>>>;
```

Comment:
This seems not relevant to this PR. Furthermore, `MeasurementData` seems not used.

Reply:
You are right. I forgot to remove it. Thanks for your pointing out.